ILSpy.WithMsiBuilder.sln

```diff
@@ -0,0 +1,135 @@
+﻿
```

Comment:
Can we add this to `ILSpy.sln` and create an `ILSpy.Installer.slnf` (solution filter)?

Reply:
Yep, just forgot to remove the file 🙈

Reply:
Done